docs/pushover.md

```
@@ -0,0 +1,53 @@
+    needs: [build, test]
+    uses: masutaka/actions/.github/workflows/pushover.yml@main
+    permissions:
+      contents: read
+    secrets:
+      PUSHOVER_API_KEY: ${{ secrets.PUSHOVER_API_KEY }}
+      PUSHOVER_USER_KEY: ${{ secrets.PUSHOVER_USER_KEY }}
+```
+
+## 関連
+
+- https://pushover.net/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)